extensions/wikia/ThemeDesigner/js/ThemeDesigner.js

```diff
@@ -1,5 +1,6 @@
-/* global $ */
-(function (window) {
+require(
+	['jquery', 'mw', 'wikia.nirvana', 'wikia.fluidlayout', 'wikia.window'],
+	function($, mw, nirvana, fluidlayout, window) {
 	'use strict';
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Use of JavaScript strict mode may result in unexpected behaviour in some browsers. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AStrictMode)